lib/web_ui/lib/src/engine/window.dart

```diff
@@ -573,8 +573,23 @@ final class EngineFlutterWindow extends EngineFlutterView implements ui.Singleto
 /// `dart:ui` window delegates to this value. However, this value has a wider
 /// API surface, providing Web-specific functionality that the standard
 /// `dart:ui` version does not.
-final EngineFlutterWindow window =
-    EngineFlutterWindow(kImplicitViewId, EnginePlatformDispatcher.instance);
+EngineFlutterWindow get window {
+  if (_window == null) {
+    throw StateError(
+      'Trying to access the implicit FlutterView, but it is not available.\n'
+      'Note: the implicit FlutterView is not available in multi-view mode.',
+    );
```

Comment:
Since the expression below needs a `!` anyway, can we make it an `assert` so the error message can be tree shaken off?

Reply:
Done